playpen/ansible/roles/dev/tasks/main.yml

```diff
@@ -55,3 +55,5 @@
 
 - name: Install custom ~/.bashrc
   copy: src=bashrc dest=/home/{{ ansible_env.SUDO_USER }}/.bashrc
+
```

Comment:
The empty line is there to separate the second copy command from the first. The "name" only applies to the first. I've been contemplating removing all uses of "name" in this and my personal playbooks because I don't think it's very valuable.